packages/utils/src/utils/agent.utils.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { HttpAgent } from "@dfinity/agent";
+import { describe, expect } from "@jest/globals";
+import {
+  mockAgentManagerConfig,
+  mockHttpAgent,
+  mockHttpAgent2,
+} from "../mocks/agent.mock";
+import { mockIdentity, mockIdentity2 } from "../mocks/identity.mock";
+import { AgentManager } from "./agent.utils";
+
+jest.mock("@dfinity/agent", () => ({
+  HttpAgent: {
+    create: jest.fn(),
+  },
+}));
+
+jest.mock("./nullish.utils", () => ({
```

Comment:
You need to mock this?

Reply:
apparently yes... when i ran the test without, it said that it couldn't find it... I will re-try and let you know

Reply:
you were right, I removed them and it works the same way